packages/api-reference/src/components/ApiReferenceLayout.test.ts

```diff
@@ -0,0 +1,65 @@
+// @vitest-environment node
+import { parse } from '@/helpers'
+import { renderToString } from '@vue/server-renderer'
+import { describe, expect, it, vi } from 'vitest'
+import { createSSRApp, h } from 'vue'
+
+import ApiReferenceLayout from './ApiReferenceLayout.vue'
+
+describe('ApiReferenceLayout', () => {
+  it('has the title in the HTML output', async () => {
+    const app = createSSRApp({
+      render: () =>
+        h(ApiReferenceLayout, {
+          configuration: {},
+          parsedSpec: {
+            info: {
+              title: 'Test API',
+            },
+          },
+          rawSpec: JSON.stringify({
+            info: {
+              title: 'Test API',
+            },
+          }),
+        }),
+    })
+
+    const html = await renderToString(app)
+
+    expect(html).toContain('Test API')
+  })
+
+  it('can render the GitHub REST API reference', async () => {
+    // Spy for console.error to avoid errors in the console
```

Comment:
I would even add warnings too

Reply:
It’ll fail then, though. Some workspace schema work ahead. 🫡

Reply:
kk just toss in a reminder somewhere to add it later!